app/src/main/java/ru/hse/spb/secretsanta/data/interfaces/DummyFestivalAdminClientInterface.kt

```diff
@@ -0,0 +1,166 @@
+                SantaGiftInfo(
+                    participantsIds[(it + 1) % participantsIds.size],
+                    SantaGiftStatus.PRESENTED
+                )
+            )
+        }.toTypedArray())
+    }
+
+    private var thisPersonWishlist: Wishlist = Wishlist(
+        listOf(
+            "Socks with different patterns and images",
+            "New PlayStation game",
+            "Interesting book",
+            "Magic wand from Harry Potter franchise (not a vibrator)",
+            "Fluffy toy",
+            "Warm sweater (my size is M)",
+            "Small backpack"
+        ),
+        listOf(
+            "Sweets",
+            "Board game",
+            "I don't like decorations. If something must stay in one place and I should look at it then it's a bad gift for me."
+        )
+    )
+
+    override fun getThisPersonWishlist(): Wishlist = thisPersonWishlist
+
+    override fun getRecipientInfo(): RecipientInfo {
+        if (mode == FestivalMode.PREPARATION) {
+            throw ClientErrorException("There is no recipient when festival has not been started.")
+        }
+
+        return RecipientInfo(
+            participantsIds[1],
+            Wishlist(
+                listOf(
+                    "Comic book",
+                    "Tasty tea",
+                    "Marzipan sweets",
+                    "Jar for cookies",
+                    "Fancy cup",
+                    "Expensive pen",
+                    "Cool poster"
+                ),
+                listOf(
+                    "Any cloth",
+                    "Be very careful if you want to give me a book. I don't like books. Not all of them, but most of them."
+                )
+            ),
+            GiftReview("That was an amazing gift! Exactly what I wanted. Thx :)")
+        )
+    }
+
+    override fun startFestival() {
+        if (mode != FestivalMode.PREPARATION) {
+            throw ClientErrorException("Festival has already been started.")
+        }
+
+        mode = FestivalMode.ACTIVE
+    }
+
+    override fun changeRules(newRules: FestivalRules) {
+        if (mode != FestivalMode.PREPARATION) {
+            throw ClientErrorException("It is not possible to change festival rules after it has been started.")
+        }
+
+        rules = newRules
+    }
+
+    override fun kickPerson(personId: PersonId) {
+    }
+
+    override fun destroyFestival() {
+    }
+
+    override fun leaveFestival() {
+    }
+
+    private var hasThisPersonSentReview: Boolean = false
+
+    override fun hasThisPersonSentReview(): Boolean {
+        return hasThisPersonSentReview
+    }
+
+    override fun sentReview(review: GiftReview?, reviewType: GiftReviewType) {
+        if (mode != FestivalMode.ENDED) {
+            throw ClientErrorException("Festival has not ended. It is not possible to send reviews.")
+        }
+
+        hasThisPersonSentReview = true
+    }
+
+    override fun changeThisPersonWishlist(newWishlist: Wishlist) {
```

Comment:
Should this become a setter of a property now?

Reply:
I wouldn't do that

Reply:
I wouldn't do that